snippets/base/admin.py

```diff
@@ -127,7 +127,11 @@ def change_view(self, request, *args, **kwargs):
         return super(BaseSnippetAdmin, self).change_view(request, *args, **kwargs)
 
     def locales(self, obj):
-        return ', '.join([locale.get_locale_display() for locale in obj.locale_set.all()])
+        locale_set =  obj.locale_set.all()[:4]
```

Comment:
I know it'd be an extra query, but I think it'd be useful to indicate how many more.

```python
num_locales = obj.locale_set.count()
locale_set = obj.locale_set.all()[:3]
active_locales = ', '.join([locale.get_locale_display() for locale in locale_set])
if num_locales > 3:
    active_locales += ', and {0} more.'.format(num_locales - 3)

return active_locales
```